_episodes_rmd/02-r-plotting.Rmd

```diff
@@ -348,25 +355,25 @@ website](https://colorbrewer2.org/) for more info about choosing plot colors.
 > > Here's an example:
 > > 
 > > ```{r Color}
-> > #install.packages("wesanderson") # install package from GitHub
-> > library(wesanderson)
-> > ggplot(data = gapminder_1997) +
-> > aes(x = gdpPercap) +
-> > labs(x = "GDP Per Capita") +
-> > aes(y = lifeExp) +
-> > labs(y = "Life Expectancy") +
-> > geom_point() +
-> > labs(title = "Do people in wealthy countries live longer?") +
-> > aes(color = continent) +
-> > scale_color_manual(values = wes_palette('Cavalcanti1'))
+> > #install.packages("ggsci")
+> > library(ggsci)
+> > ggplot(data = smoking_1990) +
+> >   aes(x = smoke_pct) +
+> >   labs(x = "Percent of people who smoke") +
+> >   aes(y = lung_cancer_pct) +
+> >   labs(y = "Percent of people with lung cancer") +
+> >   geom_point() +
+> >   labs(title = "Are lung cancer rates associated with smoking rates?") +
+> >   aes(color = continent) +
+> >   scale_color_locuszoom()
```

Comment:
For some reason I can't get the correct colors to render using this function :( Maybe we can try a different one?